app/assets/stylesheets/components/snackbars.scss

```diff
@@ -2,36 +2,32 @@
 @import '../base/boxes';
 
 .crayons-snackbar {
```

Comment:
@ludwiczakpawel has a PR for this, #7412, but I copied it over so that I could apply the new styles. Once his stuff is merged, this file will disappear from this PR.